TaskManager/TaskManager/TaskManagerProgram.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using static System.DateTime;
+
+namespace TaskManagerNamespace
+{
+    public class TaskManagerProgram
+    {
+        private static WeeklyTaskService s_service = new WeeklyTaskService();
+
+        static void Main(string[] args)
+        {
+            RunInLoop();
+        }
+
+        private static void RunInLoop()
+        {
+            string input = null;
+
+            while (input != "exit")
+            {
+                PrintMenu();
+
+                input = Console.ReadLine();
+
+                if (int.TryParse(input, out var parsedInput))
+                {
+                    HandleCommand(parsedInput);
+                }
+                else if(input == "exit")
+                {
+                    continue;
```

Comment:
we can just break here and remove condition from while: `while (true)`
```suggestion
                    break;
```